src/vue/mixins/form.mixin.js

```diff
@@ -93,9 +94,24 @@ export default {
       }
     },
     clearFields () {
-      Object.keys(this.form).forEach(key => {
-        this.form[key] = ''
-      })
+      this.clearFieldsWithOverriding({})
+    },
+    clearFieldsWithOverriding (overriddenFields) {
+      for (const [fieldName, fieldValue] of Object.entries(this.form)) {
+        if (overriddenFields[fieldName]) {
```

Comment:
you never use the method with `overriddenFields != {}`, so this `if` makes no sense

Reply:
yep, currently the only usage of this method is call in clearFields. you can use the method independently when you need to clear all the form fields but some of them to replace with another value, f.e.: you need to clear the form but not the asset selection value